libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -525,6 +526,20 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("4_ORIENT", 56, RangeFinder, state[3].orientation, ROTATION_PITCH_270),
 #endif
+
+    AP_SUBGROUPVARPTR(drivers[0], "_",  57, RangeFinder, backend_var_info[0]),
+
+#if RANGEFINDER_MAX_INSTANCES > 1
+    AP_SUBGROUPVARPTR(drivers[1], "2_", 58, RangeFinder, backend_var_info[1]),
```

Comment:
1_ ?

Reply:
@khancyr actually we don't put a "1" before the first instance's parameters so I think it's correct as-is.